code/datums/wounds/_wounds.dm

```diff
@@ -128,6 +133,16 @@
 			qdel(src)
 			return
 
+	//it's ok to not typecheck, humans are the only ones that deal with wounds
+	var/mob/living/carbon/human/human_victim = L?.owner
+	if(human_victim)
+		no_bleeding = (NOBLOOD in human_victim.dna.species.species_traits)
+		if(no_bleeding)
+			if(examine_desc_bleedless)
+				examine_desc = examine_desc_bleedless
+			if(occur_text_bleedless)
+				occur_text = occur_text_bleedless
+
```

Comment:
This is not what I meant. In apply_wounds, call a new proc called update_description() or something, and then overwrite that for the subtypes where you need to change the description

Reply:
Good call, i can do that.